samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -89,6 +98,11 @@ internal static void PrintIterationMetrics(int iteration, string trainerName, Re
             CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.RSquared ?? double.NaN,8:F4} {metrics?.MeanAbsoluteError ?? double.NaN,13:F2} {metrics?.MeanSquaredError ?? double.NaN,12:F2} {metrics?.RootMeanSquaredError ?? double.NaN,8:F2} {runtimeInSeconds.Value,9:F1}", Width);
         }
 
+        internal static void PrintIterationMetrics(int iteration, string trainerName, RankingMetrics metrics, double? runtimeInSeconds)
+        {
+            CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.DiscountedCumulativeGains.Average() ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains.Average() ?? double.NaN,9:F2} {runtimeInSeconds.Value,9:F1}", Width);
```

Comment:
I would print the NDCG@k for k=1,3,10 and DCG@k for k=10
```suggestion
            CreateRow($"{iteration,-4} {trainerName,-35} {metrics?.NormalizedDiscountedCumulativeGains[0] ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains[2] ?? double.NaN,9:F4} {metrics?.NormalizedDiscountedCumulativeGains[9] ?? double.NaN,9:F4} {metrics?.DiscountedCumulativeGains[9] ?? double.NaN,9:F4} {runtimeInSeconds.Value,9:F1}", Width);
```

These represent taking into account only the top K results, and ignoring the quality of the results beyond there. https://en.wikipedia.org/wiki/Discounted_cumulative_gain#Normalized_DCG

Reply:
Thanks for updating. 

I think you'll have to update the printed header row to match the newly added metrics. It should be approximately, "NDCG@1  NDCG@3 NDCG@10 DCG@10 RunTime" (with the right formatting of course).

Reply:
Thanks for updating the the headers.

#RESOLVED